x/incentive/migrations/v3/store.go

```diff
@@ -0,0 +1,27 @@
+package v3
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/babylonlabs-io/babylon/v4/x/incentive/types"
+)
+
+// Keeper the expected keeper interface to perform the migration
+type Keeper interface {
+	SetParams(ctx context.Context, p types.Params) error
+	GetParams(ctx context.Context) (p types.Params)
+}
+
+// MigrateStore performs in-place store migrations.
+// Migration adds the default value for the new FpPortion param
+func MigrateStore(
```

Comment:
added this migration, but in case we want to remove this and go with a gov prop after the change, [I added a nil check in the intercept_fee_collector](https://github.com/babylonlabs-io/babylon/pull/1631/files#diff-b6a48071fc2c94c8c20002a164ccf8ef1055ff5a07d5ab1d33ffd8dd58bbaeebR31)